statsmodels/discrete/tests/test_constrained.py

```diff
@@ -357,14 +357,16 @@ def setup_class(cls):
         # example with offset
         formula = 'deaths ~ smokes + C(agecat)'
         mod = GLM.from_formula(formula, data=data,
-                                    family=families.Poisson(),
-                                    offset=np.log(data['pyears'].values))
+                               family=families.Poisson(),
+                               offset=np.log(data['pyears'].values))
 
         constr = 'C(agecat)[T.4] = C(agecat)[T.5]'
         lc = patsy.DesignInfo(mod.exog_names).linear_constraint(constr)
-        cls.res1 = fit_constrained(mod, lc.coefs, lc.constants)
+
+        cls.res1 = fit_constrained(mod, lc.coefs, lc.constants,
+                                   fit_kwds={'tol': 1e-10})
         cls.constraints = lc
-        cls.res1m = mod.fit_constrained(constr)._results
+        cls.res1m = mod.fit_constrained(constr, tol=1e-10)._results
```

Comment:
it's better to be explicit and use `atol` instead of `tol` in these kinds of tests